### PR TITLE
Eraser.go changes for test

### PR DESCRIPTION
**Pull request by ashnamehrotra**
created client interface that has listImages, listContainers, and removeImage

### CODE REVIEW

File: pkg/eraser/eraser.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ spec:
     ports:
     - containerPort: 80
     imagePullPolicy: Always
+    args:  ["--runtime=containerd", "imagelist=imagelist-sample"]
```
Comment (sozercan) [inline pkg/eraser/eraser.yaml:19]: why is this hardcoded?

Comment (ashnamehrotra) [reply pkg/eraser/eraser.yaml:19]: the eraser.yaml isn't used in the project, it was just for testing deploying the pods manually

Comment (sozercan) [reply pkg/eraser/eraser.yaml:19]: aren't we automatically setting runtime value

File: pkg/eraser/eraser.go [hunk 2]
```diff
@@ -223,14 +233,40 @@ func removeVulnerableImages() (err error) {
...
-	err := removeVulnerableImages()
 
+	runtimePtr := flag.String("runtime", "containerd", "container runtime")
+	imageListPtr := flag.String("imagelist", "", "name of ImageList")
+
+	flag.Parse()
+
+	var socketPath string
+
+	// switch case
```
Comment (sozercan) [inline pkg/eraser/eraser.go:244]: let's make this a switch

**Comment by sozercan** [comment]
can we focus this on changes to eraser.go only? let's rebase to master

File: pkg/eraser/eraser.yaml (hunk 1 above)
Comment (ashnamehrotra) [reply pkg/eraser/eraser.yaml:19]: yeah but only when imagejob schedules the pods . When scheduling the pods manually for testing purposes we have to provide it

**Pull request closed without merging**